orchagent/routeorch.cpp

```diff
@@ -1896,10 +1903,10 @@ bool RouteOrch::removeRoutePost(const RouteBulkContext& ctx)
         if (it_route->second.getSize() > 1
```

Comment:
Maybe we can replace all `it_route->second` with `ol_nextHops` if this `if ... else if ...` to make use of `ol_nextHops = it_route->second`